gnovm/stdlibs/std/native.go

```diff
@@ -76,6 +76,15 @@ func X_originCaller(m *gno.Machine) string {
 }
 
 func X_originPkgAddr(m *gno.Machine) string {
+	for i := 0; i < m.NumFrames(); i++ {
+		fr := m.Frames[i]
+		if fr.LastPackage == nil || !fr.LastPackage.IsRealm() {
+			continue
+		}
+
+		return string(fr.LastPackage.GetPkgAddr().Bech32())
+	}
+
```

Comment:
@moul I'm trying to chain PreviousRealm, but I'm unsure how. Should we modify its signature and getRealm signature for iteration?

Reply:
I believe we should not allow this and should enforce that people only be aware of the direct caller. Can you explain why you would prefer otherwise?

I think that it will become common to have the pattern of expecting proxy funcs that forward their own prevrealm.

Reply:
Yes, this was actually proposed to be removed in #1475; let's go with removing instead.